src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body * {
+  box-sizing: border-box;
+  font-family: "Avenir", arial, sans-serif;
+  font-weight: 300;
```

Comment:
- always set font styles to the `html` tag
- in `*` tag only `box-sizing` can be used, anything else is forbidden

Reply:
not fixed